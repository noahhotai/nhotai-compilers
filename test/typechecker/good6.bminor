simplefunc: function void (r: integer, s: integer, t: integer ) = {

    i: integer;

    for( i=0; i<s; i++ ) {
        i: integer = 3;
        print i,"\n";
    }
}