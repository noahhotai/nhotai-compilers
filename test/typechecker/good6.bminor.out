Program parsed successfully.
i resolves to local 0
i resolves to local 0
s resolves to param 2
i resolves to local 0
i resolves to local 1
Program resolved successfully.
Program type checked successfully.
